README.md

```diff
@@ -4,10 +4,10 @@ A Python package that implements a variety of fairness-related algorithms to mit
 
 | algorithm | description | classification/regression | protected attributes | supported fairness definitions |
 | --- | --- | --- | --- | --- |
-| `expgrad` | Black-box approach to fair classification described in the paper [A Reductions Approach to Fair Classification](https://arxiv.org/abs/1803.02453)| binary classification | categorical | DP, EO |
-| `grid_search.binary_protected_attribute.binary_classification` | As described in Section 3.4 in the paper [A Reductions Approach to Fair Classification](https://arxiv.org/abs/1803.02453)| binary classification | binary | DP, EO |
-| `grid_search.binary_protected_attribute.regression` | Grid Search for regression | regression | binary | BGL |
-| `post_processing.ROCCurveBasedPostProcessing` | Post-processing algorithm based on the paper [Equality of Opportunity in Supervised Learning](https://arxiv.org/pdf/1610.02413.pdf)| binary classification | categorical | DP, EO |
+| `fairlearn.` `expgrad` | Black-box approach to fair classification described in the paper [A Reductions Approach to Fair Classification](https://arxiv.org/abs/1803.02453)| binary classification | categorical | DP, EO |
+| `fairlearn.` `reductions.` `GridSearch` | As described in Section 3.4 in the paper [A Reductions Approach to Fair Classification](https://arxiv.org/abs/1803.02453)| binary classification | binary | DP, EO |
```

Comment:
No EO yet for `GridSearch` - that's what my ongoing thread with Miro about `Moment`s etc. is (in part) about.

Reply:
Isn't EqualizedOdds pretty simple in the sense that you just pass a bunch of constraints? I thought your discussion was about regression and incorporating BGL?

Reply:
I've adjusted this to not mention EO. You can add it back if/when it's supported.